source/ota.c

```diff
@@ -1008,7 +1008,7 @@ static OtaErr_t processJobHandler( const OtaEventData_t * pEventData )
     }
 
     /* Application callback for event processed. */
-    callOtaCallback( OtaJobEventProcessed, ( const void * ) pEventData );
+    callOtaCallback( OtaJobEventProcessed, ( void * ) pEventData );
```

Comment:
No need of explicit type case to ( void * ). Same at other places in code.

Reply:
I need to keep this cast for pEventData because it's const as input, or there is a build warning.
I removed some because they're not necessary.